cmd/dockerd/config.go

```diff
@@ -65,10 +65,9 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 
 	// Deprecated flags / options
 
-	// "--graph" is "soft-deprecated" in favor of "data-root". This flag was added
-	// before Docker 1.0, so won't be removed, only hidden, to discourage its usage.
-	flags.StringVarP(&conf.Root, "graph", "g", conf.Root, "Root of the Docker runtime")
-	_ = flags.MarkHidden("graph")
+	//nolint:staticcheck // TODO(thaJeztah): remove in next release.
+	flags.StringVarP(&conf.RootDeprecated, "graph", "g", conf.Root, "Root of the Docker runtime")
```

Comment:
Oh! I'm stupid 🤦 🤦 🤦 ; I forgot to update the default for the flag; this should be

```suggestion
	flags.StringVarP(&conf.RootDeprecated, "graph", "g", conf.RootDeprecate, "Root of the Docker runtime")
```